### PR TITLE
Cdee/update grunt aws plugin

### PR DESCRIPTION
**Pull request by Chris-Dee**
Update the grunt_aws plugin to the grunt_aws_s3 plugin to allow s3 buckets to be deleted in Grunt tasks, and only deleted after build succeeds.

### CODE REVIEW

**Comment by brianlucas** [comment]
cool

**Comment by stewsmith** [comment]
@Chris-Dee Could you provide a description about how we can test?

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#Optimizely marketing website 
+#Optimizely marketing website
```
Comment (stewsmith) [inline README.md]: Is this intentional whitespace?

**Comment by Chris-Dee** [comment]
@stewsmith Since this update is just changing the add/deletion process, running and testing the staging deploys should test these changes. A failure in the aws-s3 plugin add or delete tasks should result in a build failure.

**Pull request merged**